### PR TITLE
Update known issues for 10.11.0-beta.1

### PR DESCRIPTION
**Pull request by jush**
### Summary of changes

Update known issues for `10.11.0-beta.1` release

## Pull request checklist:
 - [ ] Briefly describe the changes in this PR.
 - [ ] Include before/after visuals or gifs if this PR includes visual changes.
    
 - [ ] Write tests for all new functionality. If tests were not written, please explain why.
 - [ ] Optimize code for java consumption (`@JvmOverloads`, `@file:JvmName`, etc).
 - [ ] Add example if relevant.
 - [ ] Document any changes to public APIs.
 - [ ] Run `make update-api` to update generated api files, if there's public API changes, otherwise the `verify-api-*` CI steps might fail.
 - [ ] Update [CHANGELOG.md](../CHANGELOG.md) or use the label 'skip changelog', otherwise `check changelog` CI step will fail.
 - [ ] If this PR is a `v10.[version]` release branch fix / enhancement, merge it to `main` firstly and then port to `v10.[version]` release branch.

Fixes: < Link to related issues that will be fixed by this pull request, if they exist >

PRs must be submitted under the terms of our Contributor License Agreement [CLA](https://github.com/mapbox/mapbox-maps-android/blob/main/CONTRIBUTING.md#contributor-license-agreement).

### CODE REVIEW

**Review by yunikkk — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -32,7 +32,8 @@ Mapbox welcomes participation and contributions from everyone.
 ## Dependencies
 * Update gl-native to v10.11.0-beta.1 and common to v23.3.0-beta.1. ([1934](https://github.com/mapbox/mapbox-maps-android/pull/1934))
 
-
+## Known issues
```
Comment (yunikkk) [inline CHANGELOG.md:35]: Nit : should be also added to https://github.com/mapbox/mapbox-maps-android/releases/tag/v10.11.0-beta.1

Comment (jush) [reply CHANGELOG.md:35]: Yes, will do once we agree about the text here.

File: CHANGELOG.md [hunk 2]
```diff
@@ -32,7 +32,8 @@ Mapbox welcomes participation and contributions from everyone.
...
+* Using `BackgroundLayer.backgroundPatternTransition`, `FillExtrusionLayer.fillExtrusionPatternTransition`, `FillLayer.fillPatternTransition`, `LineLayer.lineDasharrayTransition`, `LineLayer.linePatternTransition` will result in an invalid layer.
```
Comment (ank27) [inline CHANGELOG.md:36]:
```suggestion
* Using `pattern` and `dash` transition property for layer will result in invalid layer error.
```

Comment (jush) [reply CHANGELOG.md:36]: How about combining both:
```suggestion
* Using `pattern` and `dash` transition properties for layer (e.g. `BackgroundLayer.backgroundPatternTransition`, `FillExtrusionLayer.fillExtrusionPatternTransition`, `FillLayer.fillPatternTransition`, `LineLayer.lineDasharrayTransition`, `LineLayer.linePatternTransition`, ...) will result in an invalid layer.
```

Comment (ank27) [reply CHANGELOG.md:36]: looks good 🚢

**Review by ank27 — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (jush) [reply CHANGELOG.md:35]: Done

**Pull request merged**